src/main/java/io/micronaut/gradle/graalvm/GenerateResourceConfigFile.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright 2003-2012 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.gradle.graalvm;
+
+import com.bmuschko.gradle.docker.shaded.com.fasterxml.jackson.core.util.DefaultPrettyPrinter;
+import com.bmuschko.gradle.docker.shaded.com.fasterxml.jackson.databind.ObjectMapper;
+import com.bmuschko.gradle.docker.shaded.com.fasterxml.jackson.databind.ObjectWriter;
```

Comment:
Do we want to use shaded dependencies here from the Docker plugin?

Reply:
Probably not. However I'm not super keen on adding a JSON dependency just for this, especially if we get this migrated to an official GraalVM plugin later. I would be tempted to keep this as is for the first release, then we can revisit if we don't manage to get this shipped on time, wdyt?

Reply:
I wouldn't rely on those shaded imports as Docker Java now split up their library into smaller chunks. As a result, the footprint of transitive dependencies will be smaller. In one of the latest versions of Docker Java, we started to depend on less of those dependencies from the Docker Gradle plugin. I am not sure about their long-term plans but Jackson might go away.

Reply:
Right. So given the current contents of the file I think we should just go with hand written JSON, that should be pretty straightforward.